packages/js/data/src/user/use-user-preferences.ts

```diff
@@ -118,19 +118,15 @@ export const useUserPreferences = () => {
 	// Get our dispatch methods now - this can't happen inside the callback below.
 	const dispatch = useDispatch( STORE_NAME );
 	const { addEntities, receiveCurrentUser, saveEntityRecord } = dispatch;
-	// @ts-expect-error WP 5.3.x doesn't have the User entity defined.
 	let { saveUser } = dispatch;
 
 	const userData = useSelect( ( select ) => {
 		const {
 			getCurrentUser,
 			getEntity,
 			getEntityRecord,
-			// @ts-expect-error type definition is missing.
```

Comment:
I think removing these is causing issues in the CI workflow:
<img width="1223" alt="Screen Shot 2023-03-14 at 5 31 22 PM" src="https://user-images.githubusercontent.com/10561050/225172793-dec3a8c8-dc48-42cd-be05-bad1f5c8fed7.png">

Reply:
The only reason I removed these it was because in my local env was not working with them and started to work after I removed them. I had to take another look at why locally worked but not remotely.
Thanks

Reply:
done -> [61c366d](https://github.com/woocommerce/woocommerce/pull/37196/commits/61c366da2f837278bee0a5e808c88e725b2d11ba)